services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -54,7 +63,7 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
     # CODE STATICS ---------------------------------------------------------
     API_VERSION: str = API_VERSION
     APP_NAME: str = APP_NAME
-    API_VTAG: VersionTag = parse_obj_as(VersionTag, API_VTAG)
+    API_VTAG: VersionTag = TypeAdapter(VersionTag).validate_python(API_VTAG)
```

Comment:
MINOR: in next PRs we can refactor all `ApplicationSettings` to inherit from `settings_library.application.BaseApplicationSettings`. That base class includes all env vars expected from the Dockerfile